packages/components/src/components/hds/menu-primitive/index.ts

```diff
@@ -36,6 +37,24 @@ export default class MenuPrimitiveComponent extends Component<MenuPrimitiveSigna
   @tracked toggleRef: HTMLElement | undefined;
   @tracked element!: HTMLElement;
 
+  constructor(owner: unknown, args: MenuPrimitiveSignature['Args']) {
+    super(owner, args);
+
+    deprecate(
+      'The `Hds::MenuPrimitive` component is now deprecated and will be removed in the next major version of `@hashicorp/design-system-components`.',
+      false,
+      {
+        id: 'hds.components.menu-primitive',
+        until: '5.0.0',
+        url: 'https://helios.hashicorp.design/components/menu-primitive?tab=version%20history#460',
+        for: '@hashicorp/design-system-components',
+        since: {
+          enabled: '4.8.0',
```

Comment:
At this point, it will be 4.9.0, but I'll wait until we get this reviewed and approved before committing to a version.

```suggestion
          enabled: '4.9.0',
```

Reply:
Right! updated in b8c5e9326950331f904bbf7ad52c6a26c1cde62e